catroidSourceTest/src/org/catrobat/catroid/test/code/SystemOutTest.java

```diff
@@ -38,11 +38,8 @@
 	private String errorMessages;
 	private boolean errorFound;
 
-	private static final String[] SYSTEM_OUT_DIRECTORIES = Utils.SOURCE_FILE_DIRECTORIES;
 	private static final String[] STACK_TRACE_DIRECTORIES = Utils.PRINT_STACK_TRACE_TEST_DIRECTORIES;
```

Comment:
Ohh, I found two other things ^^ The last ones, I promise!
Both variables in this line should be renamed to something ```TOAST```-related.